Error on test_traceability

ERROR openerp_test openerp.addons.stock_traceability_operation.tests.test_traceability: ` AssertionError: 3 != 2 : The wrong quants were moved: [u'344:Customers/3.000000', u'345:Customers/3.000000', u'346:Customers/4.000000']


explanation is there:
https://github.com/OCA/stock-logistics-warehouse/blob/8.0/stock_traceability_operation/tests/test_traceability.py#L160-L166

It can't works with odoo because of https://github.com/odoo/odoo/pull/10178 not merged, but works with ocb. 

I propose use exclude my module in .travis.yml for odoo but use it for OCB.

Are you ok with that ?

ping @rousseldenis

--- a/.travis.yml
+++ b/.travis.yml
@@ -21,7 +21,7 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS=1 ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - TESTS=1 ODOO_REPO="odoo/odoo" LINT_CHECK="0" EXCLUDE='stock_putaway_product'
   - TESTS=1 ODOO_REPO="OCA/OCB" LINT_CHECK="0"
 
 virtualenv:

